hooks/700-tree-skeleton.chroot

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh -ex
+# This sequence is modelled after snapd's packaging/snapd.mk it also contains
+# the important "void" directory, along with the special permissions it
+# carries.
+echo "I: Creating snapd skeleton directory structure"
+install -m 111 -d var/lib/snapd/void
```

Comment:
We have a hook-tests/ with corresponding tests for each hook, might be nice to add some basic smoke testing there.